docs/gettingStarted.md

```diff
@@ -0,0 +1,63 @@
+### Step 1 - Prerequisites
+
+Add [redux-thunk](https://github.com/gaearon/redux-thunk) middleware to your store,
+ following instructions at that repo.
+
+### Step 2 - [Config](./inputsConfig.md)
+
+Both the reducer and input props creator take a inputsConfig object that defines how inputs are initialized and validated.
```

Comment:
> Both the reducer and input props creator take...

This implies the use already knows about these, but this this the beginning of the get started. I would reword this to something like:

"The inputsConfig object defines how inputs are initialized and validated. It will be used when creating the inputs reducer and when connecting our input components."